src/data/ha-state-history-data.html

```diff
@@ -104,7 +103,9 @@
       this._setIsLoading(true);
 
       data.then(function (stateHistory) {
-        this._setData(stateHistory);
+        // Dom restoration is async and if graph is drown before dom is
```

Comment:
Actually I think `more-info-dialog` got it wrong, which can cause https://github.com/home-assistant/home-assistant-polymer/issues/60

First `isLoading` should be set to `false` so that the window will appear and then `data` should be set which will draw the graph in the already-visible window.

`more-info` does the opposite - it first sets `data` and then sets `isLoading` in a delayed way.